src/amo/pages/Guides/index.js

```diff
@@ -244,7 +244,8 @@ export const getSections = ({
           exploreMore: i18n.gettext(
             'Explore among thousands of %(linkStart)sphoto, music & video extensions%(linkEnd)s.',
           ),
-          exploreUrl: '/extensions/photos-music-videos/',
+          exploreUrl:
+            '/search/?category=photos-music-videos&sort=recommended%2Cusers&type=extension',
```

Comment:
Could you maybe use a helper function to build the query string? That would avoid to hardcode urlencoded values.

Reply:
Well, if I'm going to do that, I might as well use [`categoryResultsLinkTo`](https://github.com/mozilla/addons-frontend/blob/6dcf336e0834d98499a7ce837b37d4d3373211d7/src/amo/components/Categories/index.js#L59-L70) as well, which is probably the "right" way to do this. I'll make that change.

Reply:
Actually [`getCategoryResultsQuery`](https://github.com/mozilla/addons-frontend/blob/6dcf336e0834d98499a7ce837b37d4d3373211d7/src/core/utils/index.js#L380-L389) is more suitable, and addresses your issue. I've changed it to use that.